src/wp-admin/includes/schema.php

```diff
@@ -52,12 +52,16 @@ function wp_get_db_schema( $scope = 'all', $blog_id = null ) {
 	 */
 	$max_index_length = 191;
 
+	$enum_col     = "'" . implode( "','", array_map( 'esc_sql', wp_get_database_types() ) ) . "'";
+	$default_enum = esc_sql( wp_get_database_default_type() );
+
```

Comment:
Thinking better if these are hard-coded. This will be using the PHP data types, right? What's the point to support changing them? Option and meta values are processed in PHP.

Even if we assume that there is an edge case that might need different types, changing that from a plugin and then disabling the plugin will "wreak havoc" in a big way :)